subcommands/devices/cmd.go

```diff
@@ -27,12 +27,36 @@ var configCmd = &cobra.Command{
 }
 
 var updatesCmd = &cobra.Command{
-	Use:   "updates",
-	Short: "Device update history",
+	Use:   "updates <device> [<update-id>]",
+	Short: "Show updates performed on a device",
+	Args:  cobra.RangeArgs(1, 2),
+	Run: func(cmd *cobra.Command, args []string) {
+		if len(args) == 1 {
+			doListUpdates(cmd, args)
+		} else {
+			doShowUpdate(cmd, args)
+		}
+	},
+	Example: `
+# List all updates performed on a device:
+fioctl devices updates <device1>
+
+# List the last 2 updates:
+fioctl devices updates <device> -n2
+
+# Show the details of an update:
+fioctl devices updates <device> <update-id>
+
+# Show the most recent update with bash help:
+fioctl devices updates <device> $(fioctl devices updates <device> -n1 | tail -n1 | cut -f1 -d\ )
```

Comment:
I'd move this whole thing about updatesCmd into it's own module e.g. `updates.go`, so that this place doesn't become overcrowded.
You might even want to merge `updates_list.go` and `updates_show.go` into that module too to easily navigate things.

Reply:
that's easy enough as it gets crowded. no need yet. as we eventually deprecate, i agree that a single updates.go would be nice. for now i'm just wanting a simple improvment to this though.